ui/src/__tests__/pages/Admin/AllStories.test.ts

```diff
@@ -4,7 +4,7 @@ import {
   Action,
   allStoriesReducer,
   INIT_STATE,
-} from "../../pages/AllStories/reducer";
+} from "../../../pages/Admin/AllStories/reducer";
```

Comment:
```suggestion
} from "../../../pages/admin/AllStories/reducer";
```
is that what's breaking the test import?

Reply:
Yup fixed